tests/comprehensions.test.ts

```diff
@@ -0,0 +1,25 @@
+import { assertPrint } from "./utils.test";
+
+describe("comprehensions test", () => {
```

Comment:
Thanks for adding these! It's great that the tests from the proposal work. There are some more tests I think are important to write – the ones from your proposal are representative, but not thorough, tests.

Right now nothing uses the expression position – what about `[i * 2 for i in range(0, 5)]`?

What about creating multiple comprehensions in the same program? That would make sure nothing is global.

What about using the result of a comprehension in another comprehension? For example

```
a = [i * 2 for i in range(0, 5)]
b = [i + 1 for i in a]
```

These seem like they would be good tests of the _existing_ infrastructure that work in Python and should work with what's implemented so far.

Or what about `[print(i) for i in range(0, 5)]`, which would demonstrate that the expression is called?

Reply:
Oh ok, that makes sense @jpolitz . 

We've currently added some tests to also account for this expression part of comprehensions, but in our current implementation version, this fails (since comprehension expressions aren't currently being evaluated, as explained in my other comment).

So, once we integrate list support into comprehensions, these tests related to the evaluation of comprehension expressions should begin to pass at that stage.